probes/hasGitHubWorkflowPermissionUndeclared/def.yml

```diff
@@ -12,15 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-id: gitHubWorkflowPermissionsTopNoWrite
-short: Checks that GitHub workflows do not have default write permissions
+id: hasGitHubWorkflowPermissionUndeclared
+short: Checks that GitHub workflows have workflows without permissions declared
 motivation: >
   If no permissions are declared, a workflow's GitHub token's permissions default to write for all scopes.
   This include write permissions to push to the repository, to read encrypted secrets, etc.
   For more information, see https://docs.github.com/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token. 
 implementation: >
   The rule is implemented by checking whether the `permissions` keyword is defined at the top of the workflow,
   and that no write permissions are given.
+outcome:
+  - The probe returns 1 negative outcome per workflow without a declared permission level.
+  - The probe returns 1 OutcomeNotAvailable if if the location of the permission in the workflow is not available.
+  - The probe returns 1 OutcomeNotApplicable if the project has no workflows without declared permissions.
```

Comment:
the "motivation" section here makes me think this is about top level permissions?

Reply:
Removed in https://github.com/ossf/scorecard/pull/3816/commits/3d0d1df221b03e0747e7d8b68291ead0856c06aa